spanner/google/cloud/spanner_v1/gapic/transports/spanner_grpc_transport.py

```diff
@@ -14,8 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pkg_resources
-import grpc_gcp
```

Comment:
These imports are absolutely required:  both modules are used below in the file.

Reply:
Looks like adding the extra line broke our replacement in `synth.py`: https://github.com/googleapis/google-cloud-python/blob/ba1ddc2d54dcfa395be45dee9dd4cb35dd49c138/spanner/synth.py#L37-L47